kornia/contrib/classification.py

```diff
@@ -24,3 +26,78 @@ def __init__(self, embed_size: int = 768, num_classes: int = 10) -> None:
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         out = x.mean(-2)
         return self.linear(self.norm(out))
+
+
+def _expand(tensor, length: int):
+    return tensor.unsqueeze(1).repeat(1, int(length), 1, 1, 1).flatten(0, 1)
+
+
+class SegmentationHead(nn.Module):
```

Comment:
Why implement our own head than using torch vision models directly as in [here](https://github.com/pytorch/vision/blob/main/torchvision/models/segmentation/segmentation.py)? I would assume that we are implementing a trainer than modelling.

Reply:
There are some heads e.g. based on transformers that you cannot find yet